docker/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ubuntu:18.04
+MAINTAINER Bo Zhang (zhangb1@email.chop.edu)
+
+RUN apt-get update -y && \
+apt-get install -y python3.6 python3-pip wget git bedtools pandoc && \
+apt-get install -y build-essential libcairo2-dev libxt-dev libcurl4-gnutls-dev libxml2-dev libssl-dev libmariadbclient-dev libfontconfig1-dev libmagick++-dev libharfbuzz-dev libfribidi-dev libpq-dev
+
+#install R 3.6
+
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+RUN apt-get install -y apt-transport-https software-properties-common && \
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9 && \
+add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu bionic-cran35/' && \
+apt-get update -y && \
+apt-get install -y r-base
+
+RUN ln -s /usr/bin/python3.6 /usr/bin/python
+
+#pip install fuctions
+RUN pip3 install snakemake numpy pandas
+
+#install R packages
```

Comment:
sounds like most of those R packages were added for the RNA expression collapse module? it'd be nice we can add that description as an comment here in case we updated the module in the future, we know where to update this docker etc. for example, `dplyr` is part of the `tidyverse`, so really need to install both packages. consider the build time also each docker `RUN` adds another docker layer to instance which will make the image bigger.